makefiles/Github.toml

```diff
@@ -27,24 +27,9 @@ dependencies = [
     "build-release",
     "github-release-create-pre-release",
 ]
-command = "github-release"
-args = [
-    "-v",
-    "upload",
-    "--user", "infinyon",
-    "--repo", "fluvio",
-    "--tag", "v${FLUVIO_VERSION}",
-    "--name", "fluvio-v${FLUVIO_VERSION}-${TARGET}",
-    "--file", "./target/${TARGET}/release/fluvio",
-    "${GITHUB_RELEASE_FORCE}",
-]
-env_scripts = [
-'''
-if [ "${FORCE_RELEASE}" = "--force" ]; then
-    set_env GITHUB_RELEASE_FORCE "--replace"
-fi
```

Comment:
What did this do? Are we ok without it?

Reply:
This wasn't working right anyway. I should have removed it in #600. I think I missed it because #599 was merged too early. Anyway, the `GITHUB_RELEASE_FORCE` environment variable is set in the release.yaml.